src/stdlib/GlobalUtilities.ts

```diff
@@ -39,3 +39,23 @@ export const GetInterface = new Callable("GetInterface", {
         return object.interfaces.get(ifname.value.toLowerCase()) || BrsInvalid.Instance;
     },
 });
+
+export const FindMemberFunction = new Callable("FindMemberFunction", {
+    signature: {
+        args: [
+            new StdlibArgument("object", ValueKind.Object),
+            new StdlibArgument("funname", ValueKind.String),
+        ],
+        returns: ValueKind.Interface,
+    },
+    impl: (interpreter, object: BrsComponent, funname: BrsString): BrsInterface | BrsInvalid => {
```

Comment:
I saw your comments on the `StdLibArgument` but I think we can switch this `funname` to `funName`.